dataeng/resources/setup-platform-venv.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Install requirements
+pushd edx-platform
+make requirements
+#pip install --exists-action w -r requirements/edx/pre.txt
+#pip install --exists-action w -r requirements/edx/django.txt
+#pip install --exists-action w -r requirements/edx/base.txt
+
+# Remove if https://github.com/edx/edx-platform/pull/7465 has been merged
+#if [[ -f requirements/edx/post.txt ]]; then
+#  pip install --exists-action w -r requirements/edx/post.txt
+#fi
+
+#pip install --exists-action w -r requirements/edx/github.txt
+#pip install --exists-action w -r requirements/edx/local.txt
```

Comment:
Is there a reason to leave in these commented-out pip installs?   Are they now replaced by 'make requirements'?   I looked at the Makefile in master for edx-platform, and I only found a 'clean' target.

Reply:
Oh, my bad. Accidentally commented out.

Reply:
As, so much cleaner with all these pip installs removed :)

Reply:
Actually wait, so the latest version of the exporter I successfully got to work with a current version of the platform + a cherry picked modification from Alex.  In this new version we only need to use make requirements.  The rest of the pip install commands for edx-platform go away.

Reply:
Oh, I was looking at master.  Sigh.